shared/routes/Repl/index.html

```diff
@@ -81,23 +87,18 @@
 		100% { opacity: 1; }
 	}
 
-	.screen-too-small {
-		text-align: center;
-		margin: 0;
-		padding: 4em 3em 8em 3em;
-		line-height: 1.5;
-	}
-
 	.left, .right {
 		position: relative;
-		height: 100%;
-		float: left;
-		display: none;
 	}
 
-	.left .editor-wrapper {
-		/* make it easier to interact with scrollbar */
-		padding-right: 8px;
+	@media (max-width: 767px) {
```

Comment:
Could this be turned into a mobile first approach, i.e. defining a `min-width` for `@media`?